<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereotacticPlan</class>
 <widget class="qMRMLWidget" name="StereotacticPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>608</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="referenceToFrameCollapsibleButton">
     <property name="text">
      <string>Reference to frame</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Reference Volume: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mode: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="referenceToFrameModeComboBox">
        <item>
         <property name="text">
          <string>ACPC Align</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ACPC Register</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output Transform: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="referenceToFrameTransformNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLLinearTransformNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="editEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="calculateReferenceToFramePushButton">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="referenceVolumeLayout">
        <item>
         <widget class="qMRMLNodeComboBox" name="referenceVolumeNodeComboBox">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLScalarVolumeNode</string>
           </stringlist>
          </property>
          <property name="noneEnabled">
           <bool>true</bool>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="editEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="trajectoriesCollapsibleButton">
     <property name="text">
      <string>Trajectories</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Trajectory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="qMRMLNodeComboBox" name="trajectoryTransformNodeComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLLinearTransformNode</string>
           </stringlist>
          </property>
          <property name="noneEnabled">
           <bool>true</bool>
          </property>
          <property name="editEnabled">
           <bool>true</bool>
          </property>
          <property name="renameEnabled">
           <bool>true</bool>
          </property>
          <property name="selectNodeUponCreation">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="viewTrajectoryToolButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="resliceDriverToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="importFromToolButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mode:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="trajectoryModeComboBox">
        <item>
         <property name="text">
          <string>Target Mounting Ring Arc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Target Entry Roll</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Mounting: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="mountingComboBox">
        <item>
         <property name="text">
          <string>lateral-right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lateral-left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sagittal-anterior</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sagittal-posterior</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ring: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ctkSliderWidget" name="ringAngleSliderWidget">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Arc: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ctkSliderWidget" name="arcAngleSliderWidget">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Roll:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ctkSliderWidget" name="rollAngleSliderWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>StereotacticPlan</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>referenceToFrameTransformNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StereotacticPlan</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>referenceVolumeNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StereotacticPlan</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>trajectoryTransformNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
